### PR TITLE
[RHBMS-4312] Loading pasword from a keystore

### PR DESCRIPTION
**Pull request by rstancel**
Jira: https://issues.jboss.org/browse/RHBPMS-4312

Added loading password from a keystore also from other methods

### CODE REVIEW

File: kie-server-parent/kie-server-controller/kie-server-controller-impl/pom.xml [hunk 1]
```diff
@@ -60,6 +60,14 @@
       <groupId>com.thoughtworks.xstream</groupId>
       <artifactId>xstream</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.kie.server</groupId>
+      <artifactId>kie-server-services-common</artifactId>
```
Comment (sutaakar) [inline kie-server-parent/kie-server-controller/kie-server-controller-impl/pom.xml]: Is this dependency needed? From code changes it looks like kie-server-common should be enough.

Comment (rstancel) [reply kie-server-parent/kie-server-controller/kie-server-controller-impl/pom.xml]: You are right. Fixed

**Comment by rstancel** [comment]
Jenkins retest this

**Comment by rstancel** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
@rstancel this now has conflicts, could you please resolve

**Pull request merged**